client/src/components/ReviewListEntry.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+const moment = require('moment');
+moment().format();
```

Comment:
Line 3 can either be deleted or saved to a variable for later use in the module.